cores/arduino/api/PluggableUSB.cpp

```diff
@@ -17,85 +17,105 @@
   Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
 */
 
-#include "USBAPI.h"
```

Comment:
Modification of supposedly constant ArduinoCore-API files is not good, but I guess needed here. Flagging for the future.